tasks/update-infra-deployments/update-infra-deployments.yaml

```diff
@@ -92,7 +93,7 @@ spec:
 
         echo "original script:"
         echo "$SCRIPT"
-        PATCHED_SCRIPT=$(echo "$SCRIPT" | sed "s/{{ revision }}/$revision/g")
+        PATCHED_SCRIPT="${SCRIPT//\{\{ revision \}\}/$revision}"
```

Comment:
I'd appreciate someone checking me on this one. Here is how I tested it (with shellcheck's guidance)
```
$ var="{{ foo }} {{ foo }}"
$ foo=bar
$ echo "${var//\{\{ foo \}\}/$foo}"
bar bar
```

Reply:
Yeah, it looks correct.